browser/src/app/TilesMiddleware.ts

```diff
@@ -1554,11 +1554,10 @@ class TileManager {
 	}
 
 	private static coordsToTileBounds(coords: TileCoordData): number[] {
-		var zoomFactor = app.map.zoomToFactor(coords.z);
-		const x = (coords.x * app.pixelsToTwips) / zoomFactor;
-		const y = (coords.y * app.pixelsToTwips) / zoomFactor;
-		const width = app.tile.size.x / zoomFactor;
-		const height = app.tile.size.y / zoomFactor;
+		const x = coords.x * app.pixelsToTwips;
```

Comment:
Looks OK, though given that you return just a list of numbers here, would it make sense to make it more explicit that you take pixels here and you return twips? Perhaps rename the function to `pixelCoordsToTwipTileBounds`?

At least to me it's not too trivial that app.tile.size is in twips, but TileCoorData is in pixels.

Reply:
That's right, it confused also me when i first read the code :) Renamed.